tests/indexes/test_db.py

```diff
@@ -26,18 +27,18 @@ async def test_create(mocker, dbi, test_random_alphanumeric, static_time):
 
     mocker.patch("virtool.references.db.get_manifest", new=make_mocked_coro("manifest"))
 
-    document = await virtool.indexes.db.create(dbi, "foo", "test")
+    document = await virtool.indexes.db.create(dbi, "foo", "test", index_id=index_id)
+
+    expected_index_id = test_random_alphanumeric.history[0] if index_id is None else "abc"
+
     assert document == {
-        "_id": test_random_alphanumeric.history[0],
+        "_id": expected_index_id,
         "version": 0,
         "created_at": static_time.datetime,
         "manifest": "manifest",
         "ready": False,
         "has_files": True,
         "has_json": False,
-        "job": {
-            "id": test_random_alphanumeric.history[1]
-        },
```

Comment:
This needs to stay.

Operations:
1. Generate job ID
2. Create index document
3. Create job document and run job

Need to add `job_id` as positional argument on `virtool.indexes.db.create`.